docs/events/gsoc/2024/gradle-build-server-android.md

```diff
@@ -90,7 +90,7 @@ As the popularity of Android development grows, the need for efficient build aut
 
 ### Android Java Project Support
 
-**Status:** Complete (Under Review) :yellow_circle:
+**Status:** Complete (Merged) :green_circle:
 
 **Pull Request:** [#173](https://github.com/microsoft/build-server-for-gradle/pull/173)
```

Comment:
To provide the android support in Gradle for Java vscode extension I also added a PR in vscode-gradle repository. Should that be mentioned here? It wasn't part of the original goal though.

Reply:
It makes sense, though I would mention explicitly that it was merged after the Project end

Reply:
https://github.com/gradle/community/pull/75/commits/699ae329023fdf20c13894898ef14581b40041b3